### PR TITLE
add xcuitest migration doc (fix #6869)

### LINKED ISSUE #6869: write transition doc from UIAuto -> XCUITest iOS
We need a doc for migrating to xcuitest-based testing, that we can point people to.

### PR DESCRIPTION
**Pull request by jlipps**
add some helpful info for people new to xcuitest support. cc @imurchie

### CODE REVIEW

**Comment by jlipps** [comment]
@imurchie @yodurr I also streamlined the windows setup docs

**Comment by SrinivasanTarget** [comment]
@jlipps Nice doc. It will help many 👍

File: docs/en/advanced-concepts/migrating-to-xcuitest.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+## Migrating your iOS tests from UIAutomation (iOS 9 and below) to XCUITest (iOS 9 and up)
```
Comment (imurchie) [inline docs/en/advanced-concepts/migrating-to-xcuitest.md]: I know that technically it is iOS 9 and above, but functionally, for Appium users, it is 9.3 and above.

**Review by imurchie — COMMENTED** [review]
One small change. Otherwise looks good!

File: docs/en/advanced-concepts/migrating-to-xcuitest.md (hunk 1 above)
Comment (jlipps) [inline docs/en/advanced-concepts/migrating-to-xcuitest.md]: good point. change made!

**Pull request merged**